$OpenBSD: patch-program_include_iniset_php,v 1.3 2010/08/22 14:09:27 sthen Exp $

we set session.encrypt=0 per-directory via httpd in roundcubemail.conf.
this does not apply to upgrade.sh because it's not run by httpd, so
change to a warning rather than a fatal error so users can still use
the upgrade script without forcing session.encrypt off for the whole
server.

--- program/include/iniset.php.orig	Sun Aug 22 14:58:36 2010
+++ program/include/iniset.php	Sun Aug 22 14:58:44 2010
@@ -31,7 +31,7 @@ $crit_opts = array(
 );
 foreach ($crit_opts as $optname => $optval) {
     if ($optval != ini_get($optname)) {
-        die("ERROR: Wrong '$optname' option value. Read REQUIREMENTS section in INSTALL file or use Roundcube Installer, please!");
+        print("ERROR: Wrong '$optname' option value. Read REQUIREMENTS section in INSTALL file or use Roundcube Installer, please!");
     }
 }
 
